### DISCUSSION TITLE
Project maintenance

### DISCUSSION THREAD

**Discussion by JorgeArmandoMT** [ViroCommunity/viro#203] [General]
Hi everyone,

I hope you're all doing well. I wanted to start a discussion today about the [name of the project] project on GitHub. As you may know, this project hasn't received any updates or support in a while, despite being a really useful tool for many people.

I understand that maintaining an open-source project can be a lot of work, and I don't want to put any pressure on those who have been involved with the project in the past. However, I think it would be great if we could brainstorm some ideas for how to give this project the support it needs to keep growing and improving.

For example, maybe we could reach out to new contributors who could help with bug fixes or feature requests. Or perhaps we could explore funding options to support the ongoing maintenance of the project.

I'd love to hear your thoughts and ideas on this topic, and I'm open to suggestions on how we can move forward as a community to support this project. Thank you all for your time and consideration, and I look forward to hearing from you.

Best regards,

**Comment by cjmcassar** [comment]
Hello, I wanted to address this in light of the recent updates with the core team. We will make an announcement soon about creating a rebrand, along with a core focus. A dedicated team will be working on this open-source project full-time, and I am a part of that team. Please stay tuned for updates over the coming weeks.
